### PR TITLE
docs: add examples to documentation

### PR DESCRIPTION
**Pull request by mmcshinsky-bitgo**
Update module examples in core and utxo-lib

BG-44400

### CODE REVIEW

File: modules/utxo-lib/README.md [hunk 1]
```diff
@@ -7,24 +7,39 @@ This library is a wrapper around [bitcoinjs-lib](https://github.com/bitcoinjs/bi
...
-```typescript
+
+JavaScript (ESM)
+``` javascript
 import * as utxolib from '@bitgo/utxo-lib'
 ```
 
+NodeJS (CJS)
+``` javascript
+const utxolib = require('@bitgo/utxo-lib)'
```
Comment (tylerlevine) [inline modules/utxo-lib/README.md:28]:
```suggestion
const utxolib = require('@bitgo/utxo-lib')
```

File: modules/utxo-lib/README.md [hunk 2]
```diff
@@ -40,3 +55,4 @@ The `UtxoTransaction(Builder)` classes have the same interface as the `Transacti
 |Litecoin|`utxolib.networks.litecoin`|`utxolib.networks.litecoinTest`|
 |Zcash|`utxolib.networks.zcash`|`utxolib.networks.zcashTest`|
 
+> BitGo no longer supports sending funds to [Bitcoin SV](https://blog.bitgo.com/bsv-deprecation-6b3fff4df34c) wallets. Existing customers with said funds will still be able to access and send funds from Bitcoin SV wallets.
```
Comment (tylerlevine) [inline modules/utxo-lib/README.md:58]:
```suggestion
> [Bitcoin SV](https://blog.bitgo.com/bsv-deprecation-6b3fff4df34c) no longer supports sending funds to BitGo wallets. Existing customers with Bitcoin SV in BitGo wallets will still be able to access and sweep funds to an external Bitcoin SV wallet.
```

File: modules/utxo-lib/examples/utxo/create-address-btg.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
+/**
+ *
+ * Create various types of bitcoin gold addresses. utxolib derives from bitcoinjs-lib.
+ *
+ */
+
+{
```
Comment (tylerlevine) [inline modules/utxo-lib/examples/utxo/create-address-btg.ts:7]: why wrap in braces?

File: modules/utxo-lib/examples/utxo/create-address.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
+/**
+ *
+ * Create various types of utxo addresses. utxolib derives the following functionality from bitcoinjs-lib.
+ * Bitcoin
+ * Bitcoin Cash
+ * Bitcoin SV
+ * Litecoin
+ */
+
+{
```
Comment (tylerlevine) [inline modules/utxo-lib/examples/utxo/create-address.ts:10]: same question on braces

File: modules/bitgo/example/js/webhooks-list-notifications.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+/**
+ * Fetch a list of notifications triggers for a wallet's coin type
```
Comment (tylerlevine) [inline modules/bitgo/example/js/webhooks-list-notifications.js:2]:
```suggestion
 * Fetch a list of block notifications triggered by a webhook for a given coin type
```

File: modules/bitgo/example/js/webhooks-list-notifications.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+ * 
+ * Copyright 2019, BitGo, Inc.  All Rights Reserved.
```
Comment (tylerlevine) [inline modules/bitgo/example/js/webhooks-list-notifications.js:4]:
```suggestion
 * Copyright 2022, BitGo, Inc.  All Rights Reserved.
```

File: modules/bitgo/example/js/webhooks-simulate.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
+/**
+ * Simulate a webhook trigger for a wallet's coin type
+ * 
+ * Copyright 2019, BitGo, Inc.  All Rights Reserved.
+ */
+
+ const BitGo = require('bitgo');
```
Comment (tylerlevine) [inline modules/bitgo/example/js/webhooks-simulate.js:7]:
```suggestion
const BitGo = require('bitgo');
```

**Review by tylerlevine — CHANGES_REQUESTED** [review]
nice work, just some minor improvements from me

File: modules/utxo-lib/examples/utxo/create-address-btg.ts (hunk 3 above)
Comment (michaelmcshinsky) [reply modules/utxo-lib/examples/utxo/create-address-btg.ts:7]: I put them in as code blocks to get rid of the linting errors even though they don't actually run as part of the project was all.

**Review by tylerlevine — DISMISSED** [review]

**Review by tylerlevine — APPROVED** [review]

**Pull request merged**